iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractBranch.java

```diff
@@ -196,95 +196,109 @@ public boolean isOverloaded() {
 
     @Override
     public boolean isOverloaded(float limitReduction) {
-        return checkPermanentLimit1(limitReduction) || checkPermanentLimit2(limitReduction);
+        return checkPermanentLimit1(limitReduction, LimitType.CURRENT) || checkPermanentLimit2(limitReduction, LimitType.CURRENT);
```

Comment:
Maybe in the `Branch` API, specify in the javadoc that this method only looks at `CURRENT` type limits. Same remark for all `Overload` methods

Reply:
In fact I was wondering about adding a type parameter for those overloading checks, but don't know if it is relevant. I'll keep it like this and update the doc.